vivo-project/vivo-es
# Added initial ES configuration.

# What does this pull request do?
This PR gives provides the initial infrastructure for VIVO Elasticsearch.

# What's new?
- Dockerfile
- index-config.json
- setup-index.sh

# How should this be tested?
Follow the instructions given in `Readme.md` file. You should be able to setup ES instance to be configured either with or without security measures. You can confirm the creation of `vivo` index by visiting `http(s)://localhost:9200/vivo`.

# Interested parties
@chenejac
